### PR TITLE
Connect az updates

### PR DESCRIPTION
**Pull request by jenjoe22**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- After: https://<branch>--prisma-cloud-docs--hlxsites.hlx.page/
- Worker: https://prisma-cloud-docs-production.adobeaem.workers.dev/?branch=<branch>

### CODE REVIEW

File: docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/authorize-prisma-cloud.adoc [hunk 1]
```diff
@@ -223,19 +223,19 @@ The authentication response of the app will be returned to this URI.
 +
 The client secret is a secret string that the application uses to prove its identity when requesting a token.
 +
-.. Select menu:Certificates{sp}&{sp}secrets[+ New client secret].
+.. Select *Certificates{sp}&{sp}secrets[+ New client secret*.
```
Comment (Aishbn) [inline docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/authorize-prisma-cloud.adoc:228]: The correct syntax is *Certificates & secrets > + New client secret*.
The {sp} must be removed from the menu: format. Please fix all the instances.

File: docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/authorize-prisma-cloud.adoc [hunk 2]
```diff
@@ -277,19 +277,19 @@ image::connect/azure-account-view-roles.png[]
...
 
-* screen:[User.Read.All]
-* screen:[Policy.Read.All]
-* screen:[Group.Read.All]
-* screen:[GroupMember.Read.All]
-* screen:[Reports.Read.All]
-* screen:[Directory.Read.All]
-* screen:[Domain.Read.All]
-* screen:[Application.Read.All]
+* User.Read.All
```
Comment (Aishbn) [inline docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/authorize-prisma-cloud.adoc:282]: the screen:[User.Read.All] syntax must be replaced with `User.Read.All`.

File: docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/troubleshoot-azure-account-onboarding.adoc [hunk 3]
```diff
@@ -129,7 +129,7 @@ This query allows you to list all network traffic from the Internet or from Susp
 
 The Network Watcher is required to generate flow logs on Azure.
 
-.. Log in to the Azure portal and select menu:Network{sp}Watcher[Overview] and verify that the status is *Enabled*.
+.. Log in to the Azure portal and select *Network{sp}Watcher[Overview]* and verify that the status is *Enabled*.
```
Comment (Aishbn) [inline docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/troubleshoot-azure-account-onboarding.adoc:132]: Replace *Network{sp}Watcher[Overview]* with *Network Watcher > Overview*

File: docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/authorize-prisma-cloud.adoc (hunk 2 above)
Comment (jenjoe22) [reply docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/authorize-prisma-cloud.adoc:282]: Done

File: docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/troubleshoot-azure-account-onboarding.adoc [hunk 4]
```diff
@@ -141,7 +141,7 @@ config from cloud.resource where cloud.type = 'azure' AND api.name = 'azure-netw
 
 *Check that you have enabled flow logs on the NSGs*.
 
-.. Log in to the Azure portal, and select menu:Network{sp}Watcher[NSG Flow Logs] and verify that the status is *Enabled*.
+.. Log in to the Azure portal, and select *Network{sp}Watcher[NSG Flow Logs]* and verify that the status is *Enabled*.
```
Comment (Aishbn) [inline docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/troubleshoot-azure-account-onboarding.adoc:144]: *Network{sp}Watcher[NSG Flow Logs]* must be changed to *Network Watcher > NSG Flow Logs*

File: docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/update-azure-application-permissions.adoc [hunk 5]
```diff
@@ -1,6 +1,6 @@
 == Update Azure Application Permissions
 
-Even after you have completed onboarding, you have the option to add or remove Prisma Cloud Security Capabilities. Learn how you can add additional capabilities and the required associated permissions in your Azure subscriptions or tenants after onboarding. To verify if you have missing permissions authenticate into Prisma Cloud and select menu:Settings[Cloud Accounts], and view the *Status* column. 
+Even after you have completed onboarding, you have the option to add or remove Prisma Cloud Security Capabilities. Learn how you can add additional capabilities and the required associated permissions in your Azure subscriptions or tenants after onboarding. To verify if you have missing permissions authenticate into Prisma Cloud and select *Settings[Cloud Accounts]*, and view the *Status* column.
```
Comment (Aishbn) [inline docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/update-azure-application-permissions.adoc:3]: @jenjoe22 *Settings[Cloud Accounts]*  change to *Settings > Cloud Accounts*

File: docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/update-azure-application-permissions.adoc [hunk 6]
```diff
@@ -58,22 +58,22 @@ Follow the steps below to assign graph API permissions at the tenant level.
 
 [.procedure]
 . Add the required permissions.
-.. Navigate to the app you previously registered. Select menu:Azure{sp}Active{sp}Directory[App registrations], and select your app.
-.. Navigate to Microsoft Graph. Select menu:API{sp}permissions[+ Add a permission > Microsoft Graph > Application permissions].
+.. Navigate to the app you previously registered. Select *Azure{sp}Active{sp}Directory[App registrations]*, and select your app.
```
Comment (Aishbn) [inline docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/update-azure-application-permissions.adoc:61]: @jenjoe22 Fix menu syntax. Remove all {sp} instances in Azure topics.

File: docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/update-azure-application-permissions.adoc [hunk 7]
```diff
@@ -58,22 +58,22 @@ Follow the steps below to assign graph API permissions at the tenant level.
...
+* Reports.Read.All
+* Directory.Read.All
+* Domain.Read.All
+* Application.Read.All
 +
 If you have enabled additional functions like Agentless Scanning or Workload Protection additional permissions will be required. Review the *Roles and Permissions* list for the required permissions. 
 
 . Grant admin consent for Default Directory.
-.. Select menu:Grant{sp}admin{sp}consent{sp}for{sp}Default{sp}Directory[Yes].
+.. Select *Grant{sp}admin{sp}consent{sp}for{sp}Default{sp}Directory[Yes]*.
```
Comment (Aishbn) [inline docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/update-azure-application-permissions.adoc:77]: See my above comments.

**Review by arane26 — APPROVED** [review]
reviewed the updates

**Pull request merged**